media/css/firefox/mobile.scss

```diff
@@ -100,6 +102,7 @@ a.button.quantum.button-focus {
         margin-bottom: 0.75em;
     }
 
+    /*
```

Comment:
Nit: any reason to keep this around if it's not needed?

Reply:
D'oh!